client/src/Components/Pages/SignUp/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Button from '../../CommonComponent/Button';
+
+const Sinup = () => {
+  const data = ['Teacher', 'Parent'];
+  return (
+    <div className="container">
```

Comment:
'container' is a generic name, for the top-level element usually the component's name is being used as a class name (i.e. 'signup' in this case)